### PR TITLE
Releasenotes 2.16.0

### PR DESCRIPTION
**Pull request by rstam**

### CODE REVIEW

**Comment by rstam** [comment]
> @rstam we probably need to mention this one: https://jira.mongodb.org/browse/CSHARP-4149?focusedCommentId=4546185&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-4546185

@JamesKovacs do you want to mention anything about this one?

File: Release Notes/Release Notes v2.16.0.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+# .NET Driver Version 2.16.0 Release Notes
+
+This is the general availability release for the 2.16.0 version of the driver.
+
+The main new features in 2.16.0 include:
+
+* Support for MongoDB server version 6.0.0
+* [BETA] Support for queryable encryption
```
Comment (JamesKovacs) [inline Release Notes/Release Notes v2.16.0.md:8]: queryable encryption => Queryable Encryption
It's the official name for the new feature. So we should capitalize it.

File: Release Notes/Release Notes v2.16.0.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+* Support for creating collections with clustered indexes
+* Use count command for estimatedDocumentCount
```
Comment (JamesKovacs) [inline Release Notes/Release Notes v2.16.0.md:10]: Backticks around command names like `count` and `estimatedDocumentCount`

File: Release Notes/Release Notes v2.16.0.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+* LINQ3 bug fixes and improvements
+* Support for carry forward ($locf) and numerical/date densification
```
Comment (JamesKovacs) [inline Release Notes/Release Notes v2.16.0.md:12]: Backticks around `$locf`. And add (`$densify`) at the end.

File: Release Notes/Release Notes v2.16.0.md [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+* Support for limit in $filter expressions
```
Comment (JamesKovacs) [inline Release Notes/Release Notes v2.16.0.md:13]: Backticks around `limit` and `$filter`.

**Review by JamesKovacs — CHANGES_REQUESTED** [review]
Minor formatting changes requsted.

**Comment by JamesKovacs** [comment]
> > @rstam we probably need to mention this one: https://jira.mongodb.org/browse/CSHARP-4149?focusedCommentId=4546185&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-4546185
> 
> @JamesKovacs do you want to mention anything about this one?

That falls under the umbrella of Queryable Encryption IMHO.

**Comment by DmitryLukyanov** [comment]
> > > @rstam we probably need to mention this one: https://jira.mongodb.org/browse/CSHARP-4149?focusedCommentId=4546185&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-4546185
> > 
> > 
> > @JamesKovacs do you want to mention anything about this one?
> 
> That falls under the umbrella of Queryable Encryption IMHO.

it's specific for c# issue, but I will be ok with any decision here

**Review by JamesKovacs — APPROVED** [review]
LGTM

**Pull request merged**